src/main/host/process.c

```diff
@@ -351,8 +359,15 @@ static File* _process_openStdIOFileHelper(Process* proc, int fd, gchar* fileName
     utility_assert(fileName != NULL);
 
     File* stdfile = file_new();
+
+    char* cwd = getcwd(NULL, 0);
+    if (!cwd) {
+        error("getcwd unable to allocate string buffer, error %i: %s", errno, strerror(errno));
+    }
+
     int errcode = file_open(stdfile, fileName, O_WRONLY | O_CREAT | O_TRUNC,
-                            S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
+                            S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH, cwd);
```

Comment:
Would it make sense to make the stdout and stdin filenames relative to the process's workingDir, and then pass proc->workingDir here?

Reply:
Since we want to always create these files in the same directory regardless of the working directory of the process, I think using Shadow's working directory makes it clearer that we don't want it to be relative to the process's working directory. These paths are actually absolute paths anyways so the working directory doesn't matter in practice.